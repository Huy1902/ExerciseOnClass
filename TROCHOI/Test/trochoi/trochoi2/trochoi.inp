70
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fdchuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
chuowngtrinh123d35
abc1x2y43z
cb1
1cd7hd
jlkjhysdf122
dfslfl9204693kjlsk
1233kjlkfg3457fd
1111111111111111111dsdddddddd

