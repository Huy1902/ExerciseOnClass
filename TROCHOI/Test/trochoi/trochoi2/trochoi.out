cb1
cb1
cb1
cb1
cb1
cb1
cb1
cb1
cb1
cb1
1cd7hd
1cd7hd
1cd7hd
1cd7hd
1cd7hd
1cd7hd
1cd7hd
1cd7hd
1cd7hd
1cd7hd
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
jlkjhysdf122
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
abc1x2y43z
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
chuowngtrinh123d35
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
dfslfl9204693kjlsk
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fd
1233kjlkfg3457fdchuowngtrinh123d35
1111111111111111111dsdddddddd
